packages/wcag-contrast/docs/issue-05-luminance-compliance.md

```
@@ -0,0 +1,255 @@
+
+/**
+ * Calculate relative luminance from a color string
+ * @param color - Any supported color string
+ * @returns Relative luminance (0-1)
+ * @throws Error if color cannot be parsed
+ */
+export function relativeLuminance(color: string): number
+```
+
+**Algorithm:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)